docs/resiliency/disaster-recovery-azure-applications.md

```diff
@@ -36,31 +36,31 @@ For more information on how Azure Traffic Manager works, refer to:
+When parts of the Azure network are inaccessible, you may be unable to access your application or data. If one or more role instances are unavailable due to network issues, Azure uses the remaining available instances of your application. If your application cannot access its data because of an Azure network outage, you can potentially run with degraded application functionality locally by using cached data. You need to design the disaster recovery strategy to run with degraded functionality in your application. For some applications, this might not be practical.
 
-Another option is to store data in an alternate location until connectivity is restored. If degraded mode is not an option, the remaining options are application downtime or failover to an alternate region. The design of an application running in degraded mode is as much a business decision as a technical one. This is discussed further in the section on [degraded application functionality](#degraded-application-functionality).
+Another option is to store data in an alternate location until connectivity is restored. If degrading functionality is not an option, the remaining options are application downtime or failover to an alternate region. The design of an application running with degraded functionality is as much a business decision as a technical one. This is discussed further in the section on [degraded application functionality](#degraded-application-functionality).
 
 ### Failure of a dependent service
-Azure provides many services that can experience periodic downtime. Consider [Azure Redis Cache](https://azure.microsoft.com/services/cache/) as an example. This multi-tenant service provides caching capabilities to your application. It's important to consider what happens in your application if the dependent service is unavailable. In many ways, this scenario is similar to the network outage scenario. However, considering each service independently results in potential improvements to your overall plan.
+Azure provides many services that can experience periodic downtime. For example, [Azure Redis Cache](https://azure.microsoft.com/services/cache/) is a multi-tenant service which provides caching capabilities to your application. It's important to consider what happens in your application if the dependent service is unavailable. In many ways, this scenario is similar to the network outage scenario. However, considering each service independently results in potential improvements to your overall plan.
```

Comment:
Cache is actually a weird example to use here. If the cache is unreachable, it should never be a DR issue almost by definition (that's why it's a cache...)